src/components/NewChapterForm.tsx

```diff
@@ -226,8 +242,18 @@ const NewChapterForm = () => {
           <div className="grid place-items-center pt-8">
             <input
```

Comment:
If an error occurs or the user retype after submitting, should we reset the state? `setFormSubmission(not submitted)`